tests/strategy/helpers.py

```diff
@@ -0,0 +1,9 @@
+from tests.factories.rest.market import OfferProposalFactory
+
+
+def mock_offer(provider_id=1, coeffs=(0.001, 0.002, 0.1)):
```

Comment:
the only thing is that I'd consider adding a method on the `OfferProposalFactory` instead of creating a separate `mock_offer` function...

Reply:
or, just override the `generate` method of `OfferProposalFactory` to accept additional `provider_id` and `coeffs` parameters

Reply:
c91bc6a